docs/scenarios/parametric.md

```diff
@@ -202,41 +222,22 @@ There is two ways for running the C++ library tests with a custom tracer:
     * `https://github.com/DataDog/dd-trace-cpp@<COMMIT HASH>`
 2. Clone the dd-trace-cpp repo inside `binaries`
 
-The parametric shared tests can be run against the C++ library,
-[dd-trace-cpp][1], this way:
-```console
-$ TEST_LIBRARY=cpp ./run.sh PARAMETRIC
-```
-
-Use the `-k` command line argument, which is forwarded to [pytest][2], to
-specify a substring within a particular test file, class, or method. Then only
-matching tests will run, e.g.
-```console
-$ TEST_LIBRARY=cpp ./run.sh PARAMETRIC -k test_headers
-```
-
-It's convenient to have a pretty printer for the tests' XML output. I use
-[xunit-viewer][3].
-```console
-$ npm install junit-viewer -g
-```
-
-My development iterations then involve running the following at the top of the
-repository:
-```console
-$ TEST_LIBRARY=cpp ./run.sh PARAMETRIC -k test_headers; xunit-viewer -r logs_parametric/reportJunit.xml
+#### When you are done testing against a custom tracer:
+```bash
+rm -rf binaries/python-load-from-pip
 ```
 
-This will create a file `index.html` at the top of the repository, which I then
-inspect with a web browser.
-
-The C++ build can be made to point to a different GitHub branch by modifying the
-`FetchContent_Declare` command's `GIT_TAG` argument in [CMakeLists.txt][4].
+### Understanding the test outcomes
```

Comment:
Cold you add only a ling to `docs/execute/test-outcomes.md` here ? I'd prefer not to duplicate information in the doc.